libs/core/src/storage/noopstorage/noopstorage.service.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { DaffNoopStorageService } from './noopstorage.service';
+
+fdescribe('DaffNoopStorageService', () => {
+  let service: DaffNoopStorageService;
+
+  beforeEach(() => {
+    service = new DaffNoopStorageService;
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+  
+  it('should not be able to set a value in memory', () => {
+		service.setItem('testkey', 'testvalue');
+
+		expect(service.getItem('testkey')).toEqual(undefined);
+	});
+	
+	it('should not be able to retrieve a value from memory', () => {
+    service.setItem('testkey', 'testvalue');
+
+		expect(service.getItem('testkey')).toEqual(undefined);
+	});
+	
+	it('should be able to delete an item from memory', () => {
```

Comment:
I don't understand this behavior. I would think that a noop service should not be interacting with memory at all.

Reply:
@damienwebdev